tests/basic/clone/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.6 AS builder
```

Comment:
out of curiosity, given that 3.6 is rather dated. is there a "no later than X" dependency for sgx-lkl that I'm not aware of?

Reply:
We should use a consistent Alpine version in all tests. @letmaik?

Reply:
I opened https://github.com/lsds/sgx-lkl/issues/260. For this PR, it's fine to leave it at an arbitrary version. The [maximum supported](https://github.com/lsds/sgx-lkl/issues/200) currently is 3.10.